app/src/main/java/org/mifos/mobilebanking/ui/fragments/HomeOldFragment.java

```diff
@@ -132,11 +133,16 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
         MenuItem menuItem = menu.findItem(R.id.menu_notifications);
         View count = menuItem.getActionView();
         tvNotificationCount = (TextView) count.findViewById(R.id.tv_notification_indicator);
+        presenter.getUnreadNotificationsCount();
         count.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                ((BaseActivity) getActivity()).replaceFragment(NotificationFragment.newInstance(),
-                        true, R.id.container);
+
+                startActivity(
```

Comment:
Please change this into 1 line or 2, no need of intending it in this way

Reply:
Ok